src/commentcnv.l

```diff
@@ -436,14 +444,6 @@ SLASHopt [/]*
 				     yyextra->readLineCtx=YY_START;
 				     BEGIN(ReadLine);
                                    }
-<Scan>{CPPC}[!/]/.*\n              { /* one line special C++ comment */
```

Comment:
@albert-github can this rule really be removed? Seems like any case that is not a `/*! @}` will now match the rule below, treating it as a normal comment and missing any special commands inside it? or is this then still handled elsewhere?

Reply:
@doxygen 
Not sure about it. I expected to get a warning about something like a double rule, but that is not the case so probably it doesn't hurt to leave those lines.

Should I update the PR or will you do it upon integration?